### PR TITLE
Anna proofread

### PR DESCRIPTION
**Pull request by grantbacon-jaspersoft**

### CODE REVIEW

**Comment by grantbacon-jaspersoft** [comment]
LGTM!

File: pages/input-controls/override-styles.md [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
 
 The package `@jaspersoft/jv-ui-components` provides a stylesheet that contains the default styles for the input controls.
-This file can be located in the following path: `@jaspersoft/jv-ui-components/dist/jv-ui.css`.
-By default, it is imported in the sample app's `index.css` and `App.tsx` files. E.g:
+This file can be located at the following path: `@jaspersoft/jv-ui-components/dist/jv-ui.css`.
+It is imported in the sample application's `index.css` and `App.tsx` files as follows:
 ```css
 @import "@jaspersoft/jv-ui-components/dist/jv-ui.css";
 ```
+Optionally, there is a configuration that enables you to use the default styles for the input controls components only. When this configuration is used, the other styles in your application will not be affected by the Visualize Components CSS:
```
Comment (ecanchev-jaspersoft) [inline pages/input-controls/override-styles.md]: could we remove the word '`optionally`'?
this isn't optional, it is mandatory to apply the proper styling of our components.

File: pages/scheduler/basic-usage.md [hunk 2]
```diff
@@ -74,40 +71,35 @@ import {
...
-    schedulerUIConfig,
-);
-
-```
-
-* `container`: (DivElement) <div> element from the DOM where the scheduler should be rendered
-* `visualize_object`: (VisualizeClient) visualization object
-* `schedulerUIConfig`: (SchedulerConfig) object containing configuration for scheduler look and feel and event handling. [reference]({{site.baseurl}}/pages/scheduler/configuration.html#configuration)
+Scheduler package provides a
+`renderScheduler(v: VisualizeClient, container: HTMLElement, config: SchedulerConfigProps): void` method.
```
Comment (ecanchev-jaspersoft) [inline pages/scheduler/basic-usage.md]: This method is missing the '`uri:string`' parameter.
It has been added recently to match the same interface as in `InputControls` package.

**Review by ecanchev-jaspersoft — CHANGES_REQUESTED** [review]

File: pages/scheduler/basic-usage.md (hunk 2 above)
Comment (yplakosh-jaspersoft) [reply pages/scheduler/basic-usage.md]: Done

File: pages/input-controls/override-styles.md (hunk 1 above)
Comment (yplakosh-jaspersoft) [reply pages/input-controls/override-styles.md]: Done.

**Review by ecanchev-jaspersoft — APPROVED** [review]
Approved

**Pull request merged**